pages/dkp/kommander/1.4/release-notes/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Kommander v1.4 Release Notes
+title: Release Notes
+menuWeight: 0
+excerpt: View release-specific information for Kommander
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on ?????.**
+
+[button color="purple" href="https://support.d2iq.com/s/entitlement-based-product-downloads"]Download Konvoy[/button]
+
+To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [install](/dkp/konvoy/latest/install/) the latest version of Konvoy.
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install Konvoy.</p>
+
+# Release Summary
+Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated cost management across multiple clusters, whether they are a new Konvoy cluster or an existing 3rd party/DIY distribution installation.
+
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.19.7  |
+
+# Breaking changes
+
+## Docker hub rate limiting 
+Docker Hub announced an update to their image pull policies in August, 2020. The change results in the need to change cluster configurations to accommodate new account structures that enable image pull rate limiting.
```

Comment:
I think this was copied from 1.3 and doesn't apply to 1.4 anymore

Reply:
it does still apply.